jupytext/header.py

```diff
@@ -144,7 +144,7 @@ def header_to_metadata_and_cell(lines, header_prefix, ext=None):
     encoding_re = re.compile(r'^[ \t\f]*{}.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)'.format(comment))
```

Comment:
Regarding the encoding, do you know it it should use the language line comment, or `#`?

Reply:
I only know the encoding line from Python 2 programs, so I am not sure if other languages use this feature as well.